tests/WorkOS/PortalTest.php

```diff
@@ -12,10 +12,36 @@ protected function setUp()
     {
         $this->traitSetUp();
 
-        $this->withApiKeyAndProjectId();
+        $this->withApiKey();
         $this->ap = new Portal();
     }
 
+    public function testCreateOrganization()
+    {
+        $organizationsPath = "organizations";
+
+        $result = $this->createOrganizationResponseFixture();
+
+        $params = [
+            "name" => "Organization Name",
+            "domains" => array("example.com")
+        ];
+
+        $this->mockRequest(
+            Client::METHOD_POST,
+            $organizationsPath,
+            null,
+            $params,
+            true,
+            $result,
```

Comment:
I think older versions of PHP are throwing up because of this trailing comma /shrug